### ISSUE TITLE
Request to display directions N/S/E/W on Target

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenPHDGuiding/phd2#347]
```
Post from Pascal:

some months ago I suggested to Andy to add  the cardinal directions (N S E W) 
on the PHD graphs in order to quickly help to identify guiding problems (polar 
alignment,  drifting, balancing etc). If I remember the problem was the 
knowledge of the pier side to properly identify the north and south, but now 
with the 'aux mount' function (great think, thanks!) may be this is no more a 
problem. May be you can reconsider the possibility to add the four useful 
letters in the PHD graphs? 

thread: 
https://groups.google.com/forum/?fromgroups=#!topic/open-phd-guiding/qOqLkmHnWno

```

Original issue reported on code.google.com by `andy.gal...@gmail.com` on 19 Oct 2014 at 4:57

**Comment by agalasso** [comment]
feature was implemented in 6dc50d4262456636f3ab5bd4c63e40e787295323 including adding labels on the RA/Dec overlay

leaving the issue open until we implement the labels on the target and the history graph

**Comment by agalasso** [comment]
target display now shows the sky directions 1e2de1e954d2b6d63a2392f82ce24727e5264416

we're not going to show the sky directions on the graph since this will introduce unnecessary confusion about sky directions vs mount directions, especially when it comes to deciding which direction to choose for uni-directional Dec Guide Mode.  Mount directions are now labeled on the graph.
